packages/cli/src/serve/output.rs

```diff
@@ -159,7 +162,7 @@ impl Output {
     /// Why is the ctrl_c handler here?
     ///
     /// Also tick animations every few ms
-    pub async fn wait(&mut self) -> io::Result<()> {
+    pub async fn wait(&mut self) -> io::Result<bool> {
```

Comment:
In the future we should provide a comment as to what the boolean actually means - or communicate this through a different mechanism.